hack/generator/pkg/astmodel/json_type.go

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astmodel
+
+// APIExtensions is the package containing the type we use to
+// represent arbitrary JSON fields.
+const APIExtensions = "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
Consider changing this to a `PackageReference`
```suggestion
var APIExtensionsReference PackageReference = NewExternalPackageReference("k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1")
```

Reply:
Yup, I've done this.